client/js/libs/handlebars/tz.js

```diff
@@ -2,6 +2,6 @@
 
 const moment = require("moment");
 
-module.exports = function(time) {
-	return moment(time).format("HH:mm");
+module.exports = function(time, seconds) {
+	return seconds ? moment(time).format("HH:mm:ss") : moment(time).format("HH:mm");
```

Comment:
Can I be a code pedant and request something like this:
```
const format = seconds ? "HH:mm:ss" : "HH:mm";
return moment(time).format(format);
```

I would also prefer if the formats were in constants somewhere, but I won't push that one too hard.